solutions/USACO jan19-grass-planting.mdx

```diff
@@ -72,4 +74,67 @@ public class GrassPlanting {
 
 </JavaSection>
 
-</LanguageSection>
\ No newline at end of file
+</LanguageSection>
+
+## Solution 2 - DFS
+
+**Time Complexity**: $\mathcal{O}(N)$
+
+This can also be solved using DFS. The solution is less elegant but generates a valid assignment of grass types in the $\texttt{deg}$ array. 
```

Comment:
```suggestion
This can also be solved using DFS. The solution is less elegant but generates a valid assignment of grass types in the $\texttt{ans}$ array. 
```